FirebaseRemoteConfig/SwiftNew/ConfigRealtime.swift

```diff
@@ -496,7 +500,13 @@ class ConfigRealtime: NSObject, URLSessionDataDelegate {
           evaluateStreamResponse(response)
         }
       } catch {
-        propagateErrors(error)
+        let wrappedError = NSError(domain: RemoteConfigUpdateErrorDomain,
```

Comment:
For future reference, this is the code that likely needs to be rewritten to solve https://github.com/firebase/firebase-ios-sdk/issues/14094